recsys/recsys_tutorial.py

```diff
@@ -111,7 +134,7 @@ def stars_in_review(x):
 
 @preprocessor(memoize=True)
 def textblob_polarity(x):
-    if x.review_text:
+    if isinstance(x.review_text, str):
         x.blob = TextBlob(str(x.review_text))
```

Comment:
Don't need the str cast. Actually, I don't remember why I had that cast in the first place, since non-None review_text's should always be strings

Reply:
One of them is the integer 70